orchagent/pfcactionhandler.cpp

```diff
@@ -593,7 +593,7 @@ void PfcWdZeroBufferHandler::ZeroBufferProfile::createZeroBufferProfile(bool ing
     attribs.push_back(attr);
 
     attr.id = SAI_BUFFER_PROFILE_ATTR_SHARED_DYNAMIC_TH;
-    attr.value.u32 = 1;
+    attr.value.u32 = -8; // ALPHA_0
```

Comment:
@andriymoroz-mlnx This usage of alpha == -8, indicating zero shared pool, is MLNX proprietary, not defined/standardized in SAI SAI_BUFFER_PROFILE_ATTR_SHARED_DYNAMIC_TH